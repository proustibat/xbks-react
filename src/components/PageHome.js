import React from 'react';

const PageHome = () => (
    <div>
        <h1>Hello XBKS</h1>
    </div>
);

export default PageHome;
