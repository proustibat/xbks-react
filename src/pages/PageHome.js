import React from 'react';
import BookList from '../components/BookList';

const PageHome = () => (
    <BookList />
);

export default PageHome;
