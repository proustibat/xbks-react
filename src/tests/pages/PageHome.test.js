import React from 'react';
import { shallow } from 'enzyme';
import PageHome from '../../pages/PageHome';

test( 'Should render home page correctly', () => {
    const wrapper = shallow( <PageHome /> );
    expect( wrapper ).toMatchSnapshot();
} );
