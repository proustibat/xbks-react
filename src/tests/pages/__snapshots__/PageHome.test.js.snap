// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render home page correctly 1`] = `
<div>
  <Connect(SearchBar) />
  <Connect(BookList) />
</div>
`;
